dapr/clients/grpc/client.py

```diff
@@ -1088,6 +1088,226 @@ def unlock(
+            workflow_component=workflow_component)
+        response, call = self._stub.GetWorkflowAlpha1.with_call(req)
+
+        # RRL TODO: Fix this
+        return api_v1.GetWorkflowResponse(instance_id=instance_id,
+                                          workflow_name=response.workflow_name,
+                                          created_at=response.created_at,
+                                          last_updated_at=response.last_updated_at,
+                                          runtime_status=response.runtime_status)
+
+    def terminate_workflow(
+            self,
+            instance_id: str,
+            workflow_component: str) -> DaprResponse:
+        """Terminates a workflow.
+
+            Args:
+                instance_id (str): the name of the workflow instance, e.g.
+                                    `order_processing_workflow-103784`.
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+
+            Returns:
+                :class:`DaprResponse` gRPC metadata returned from callee
+
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component)
+        # Actual terminate workflow invocation
+        req = api_v1.TerminateWorkflowRequest(
+            instance_id=instance_id,
+            workflow_component=workflow_component)
+        _, call = self._stub.TerminateWorkflowAlpha1.with_call(req)
+
+        return DaprResponse(
+            headers=call.initial_metadata())
+
+    # RRL TODO: Clean up return obkect comments
+    def raise_event(
+            self,
+            instance_id: str,
+            workflow_component: str) -> DaprResponse:
+        """Raises an event on a workflow.
+
+            Args:
+                instance_id (str): the name of the workflow instance,
+                                    e.g. `order_processing_workflow-103784`.
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+
+            Returns:
+                :class:`DaprResponse` gRPC metadata returned from callee
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component)
+        # Actual terminate workflow invocation
+        req = api_v1.TerminateWorkflowRequest(
+            instance_id=instance_id,
+            workflow_component=workflow_component)
+        _, call = self._stub.TerminateWorkflowAlpha1.with_call(req)
```

Comment:
Why does this also call TerminateWorkflow? I thought Raise Event is something else?